src/main/java/io/appform/dropwizard/actors/base/Handler.java

```diff
@@ -58,12 +70,19 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                                AMQP.BasicProperties properties, byte[] body) throws IOException {
         try {
             final Message message = mapper.readValue(body, clazz);
-            boolean success = retryStrategy.execute(() -> handle(message, messageProperties(envelope)));
+            val tracer = GlobalTracer.get();
```

Comment:
create a separate method for tracing requirements

Reply:
tracing should be behind a config. By default, tracing will be disabled which is the current behaviour.

Reply:
created separate methods and put tracing behind config for each actor